lib/impl/AwsS3Tvm.js

```diff
@@ -99,15 +99,18 @@ class AwsS3Tvm extends Tvm {
       secretAccessKey: params.awsSecretAccessKey
     })
 
+    const s3Folder = params.owNamespace // allowed chars match
```

Comment:
We don't use this `const s3Folder` anywhere except one place. So we can just use `params.owNamespace` directly and avoid this extra change.

Reply:
thanks for the suggestion @sandeep-paliwal.
But. I would like to have the code as clear as possible on this. let's keep the extra variable